php-checks/src/test/resources/checks/phpunit/AssertTrueInsteadOfDedicatedAssertCheck.php

```diff
@@ -29,9 +29,22 @@ public function test()
 
     assertTrue(null == null); // Noncompliant
 
-    assertTrue(strpos($this->Mail->Body, 'src="composer.json"') === false); // Noncompliant {{Use assertSame instead.}}
-    assertTrue(strpos($this->Mail->Body, 'src="composer.json"') !== false); // Noncompliant {{Use assertNotSame instead.}}
-    assertTrue(doSomeThing() === true); // Noncompliant
+    assertTrue(strpos($this->Mail->Body, 'src="composer.json"') === false); // Noncompliant {{Simplify this expression by removing the equality comparison to false and use assertFalse instead.}}
+    assertTrue(strpos($this->Mail->Body, 'src="composer.json"') !== false); // Noncompliant {{Simplify this expression by removing the non-equality comparison to false.}}
+
+    assertTrue ($a == true);  // Noncompliant {{Simplify this expression by removing the equality comparison to true.}}
+    assertTrue ($a == false); // Noncompliant {{Simplify this expression by removing the equality comparison to false and use assertFalse instead.}}
```

Comment:
I find it more concise to write:
`Simplify this expression by using assertFalse() and removing the comparison to "false"`
IMO I would drop the words `equality` and `non-equality`